guide/14-deep-learning/how_AutoDL_works.ipynb

```diff
@@ -0,0 +1,566 @@
+{
```

Comment:
<p>Reword:</p><p>This guide explains the steps for training and evaluating multiple network performance supported by <code>arcgis.learn</code> API. </p><p>to </p><p>This guide explains the steps for training and evaluation of multiple network architectures supported by <code>arcgis.learn</code> API.</p><p> </p><p>comma is needed after <code>more specifically for object detetcion task only</code></p><p> </p><p>comma is needed after 'train a deep learning network using the <code>arcgis.learn</code> API'</p><p> </p><p>remove 'and' from <code>AutoDL class automatically trains all the supported network with the given data and</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1198/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>